PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -674,6 +733,7 @@ - (instancetype)initForClient:(PubNub *)client {
         _presenceChannelsSet = [NSMutableSet new];
         _resourceAccessQueue = dispatch_queue_create("com.pubnub.subscriber",
                                                      DISPATCH_QUEUE_CONCURRENT);
+        _escapedFilterExpression = [PNString percentEscapedString:client.filterExpression];
```

Comment:
@jzucker2 If you check subscriber's code, you will see what it is normal thing to access client's configuration, so there is no need in _proxy_ method from _PubNub_ class to get filter expression. Use direct access to configuration's field.